### PR TITLE
feat: add vertical and horizontal offsets to the popover component

### PR DESCRIPTION
**Pull request by sylvanasantos**
### Overview

- Replaced offset prop on Popover component with verticalOffset and horizontalOffset 
- These two new props will allow clients to shift the Popover vertically and horizontally from the source

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: EGG-311
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by sylvanasantos** [comment]
sorry! meant to make this a draft! not quite ready, will ping when it is :)

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.5.6-alpha.58eb93.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fdb759ac2e5bf16e495ebe4--gamut-preview.netlify.app>https://5fdb759ac2e5bf16e495ebe4--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fdb759ac2e5bf16e495ebe4>Deploy Logs</a></p>

**Comment by sylvanasantos** [comment]
okay ready for review!

File: packages/gamut-labs/src/experimental/Popover/index.tsx [hunk 1]
```diff
@@ -67,18 +72,18 @@ export const Popover: React.FC<PopoverProps> = ({
...
-      above: Math.round(targetRect.top - offset),
-      below: Math.round(targetRect.top + targetRect.height + offset),
+      above: Math.round(targetRect.top - verticalOffset),
+      below: Math.round(targetRect.top + targetRect.height + verticalOffset),
     };
     const alignments = {
-      right: Math.round(window.scrollX + targetRect.right),
-      left: Math.round(window.scrollX + targetRect.left),
+      right: Math.round(window.scrollX + targetRect.right + horizontalOffset),
+      left: Math.round(window.scrollX + targetRect.left + horizontalOffset),
```
Comment (hmafzal) [inline packages/gamut-labs/src/experimental/Popover/index.tsx:80]: I think the way offset originally worked is: how much padding away from target rect to offset (see how `above` is `- offset` and `below` is `+ offset`

I think for consistency maybe this should be `- horizontalOffset`

**Review by hmafzal — APPROVED** [review]
LGTM

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@7.5.6-alpha.a0b1e5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5fdbed6fac704500c6c25811--gamut-preview.netlify.app>https://5fdbed6fac704500c6c25811--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5fdbed6fac704500c6c25811>Deploy Logs</a></p>

**Pull request merged**